src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Rename to singular `Colors` -> `Color`
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html